### ISSUE TITLE
pixel and mask encoder were not be trained

### ISSUE THREAD

**Issue by zhangming8** [hkchengrex/Cutie#135]
Hi, I found you override the "train()" function in mask encoder and pixel encoder:
https://github.com/hkchengrex/Cutie/blob/main/cutie/model/big_modules.py#L57
https://github.com/hkchengrex/Cutie/blob/main/cutie/model/big_modules.py#L185

This means that the params of mask encoder and pixel encoder were freeze during all your training? even though you set param groups in https://github.com/hkchengrex/Cutie/blob/main/cutie/model/utils/parameter_groups.py#L56

**Comment by hkchengrex** [comment]
That means the layers (i.e., BN) are set to `eval` mode. They are still being trained. See https://discuss.pytorch.org/t/model-eval-vs-with-torch-no-grad/19615

**Comment by zhangming8** [comment]
> That means the layers (i.e., BN) are set to `eval` mode. They are still being trained. See https://discuss.pytorch.org/t/model-eval-vs-with-torch-no-grad/19615

Yes, you are right.
I compared the encoder's parameters before and after training, and they were all different except for bn's running_mean and running_var.
